### ISSUE TITLE
One of the config watcher threads wakes up every 100ms

### ISSUE THREAD

**Issue by WGH-** [jwilm/alacritty#2029]
Not a good thing for powersaving. Disabling `live_config_reload` disables the problematic thread, too.

Caused by this: passcod/notify#173

```
strace: Process 4291 attached
epoll_wait(10, [], 16, 62)              = 0
epoll_wait(10, [], 16, 100)             = 0
epoll_wait(10, [], 16, 100)             = 0
epoll_wait(10, [], 16, 100)             = 0
epoll_wait(10, [], 16, 100)             = 0
epoll_wait(10, [], 16, 100)             = 0
```

**Comment by nixpulvis** [comment]
I can indeed replicate this issue, and the upstream PR (passcod/notify#174) does fix this issue, while still allowing config changes to be picked up. Comparing the output of these two `strace` calls with a local change to our `Cargo.toml` to use the upstream PR.

```
strace -fe epoll_wait alacritty
strace -fe epoll_wait cargo run
```

**Comment by chrisduerr** [comment]
Should be resolved by #2035.
